test/regression.uts

```diff
@@ -266,10 +266,6 @@ Ether(dst="ff:ff:ff:ff:ff:ff", src="ff:ff:ff:ff:ff:ff", type=0x9000)
 assert _.mysummary() in ['ff:ff:ff:ff:ff:ff > ff:ff:ff:ff:ff:ff (%s)' % loop
                          for loop in ['0x9000', 'LOOP']]
 
-= Test fletcher16_* functions
-assert(fletcher16_checksum(b"\x28\x07") == 22319)
-assert(fletcher16_checkbytes(b"ABCDEF", 2) == "\x89\x67")
```

Comment:
Hold on, it seems that simply using
`assert(fletcher16_checkbytes(b"ABCDEF", 2) == b"\x89\x67")`
fixes it

Reply:
Oh, you're right ^^